packages/features/eventtypes/components/tabs/recurring/RecurringEventController.tsx

```diff
@@ -2,22 +2,44 @@ import { useState } from "react";
 import { useFormContext } from "react-hook-form";
 
 import useLockedFieldsManager from "@calcom/features/ee/managed-event-types/hooks/useLockedFieldsManager";
-import type { EventTypeSetup } from "@calcom/features/eventtypes/lib/types";
+import type {
+  EventTypeSetup,
+  InputClassNames,
+  SelectClassNames,
+  SettingsToggleClassNames,
+} from "@calcom/features/eventtypes/lib/types";
 import type { FormValues } from "@calcom/features/eventtypes/lib/types";
 import { classNames } from "@calcom/lib";
 import { useLocale } from "@calcom/lib/hooks/useLocale";
 import { Frequency } from "@calcom/prisma/zod-utils";
 import type { RecurringEvent } from "@calcom/types/Calendar";
 import { Alert, Select, SettingsToggle, TextField } from "@calcom/ui";
 
-type RecurringEventControllerProps = {
+export type RecurringEventControllerProps = {
   eventType: EventTypeSetup;
   paymentEnabled: boolean;
+  customClassNames?: EventRecurringTabCustomClassNames;
+};
+
+export type EventRecurringTabCustomClassNames = {
+  container?: string;
+  recurringToggle?: SettingsToggleClassNames;
+  frequencyInput?: InputClassNames;
+  frequencyUnitSelect?: SelectClassNames;
+  maxEventsInput?: {
```

Comment:
In the API we define it as `ocurrenceCount`. I would name it like it because we already have a nested `countInput` and `maxEventsInput` sounds like we will style individual input not the whole section.